gg-fleet-statusd/bin/gg-fleet-statusd.c

```diff
@@ -4,6 +4,10 @@
 
 #include "gg_fleet_statusd.h"
 #include <ggl/error.h>
+#include <ggl/version.h>
+
+__attribute__((visibility("default"))) const char *argp_program_version
```

Comment:
Why do you need this?

Reply:
If you do `./bin/gg-fleet-statusd --version` you will get the same value as nucleus